dev/WindowsAppRuntime_BootstrapDLL/MddBootstrap.h

```diff
@@ -25,6 +25,48 @@ STDAPI MddBootstrapInitialize(
     PCWSTR versionTag,
     PACKAGE_VERSION minVersion) noexcept;
 
+/// Options for MddBootstrapInitialize()
+typedef enum MddBootstrapInitializeOptions
```

Comment:
we should add another enum to silently ignore failure in packaged apps (for Fred)

Reply:
works for me

Reply:
`MddBootstrapInitializeOptions_OnPackageIdentity_NOP` for clarity 

This avoids the whole 'packaged'/'unpackaged' app lingo, 'is a sparse package a packaged app' and like questions. Merely the statement of fact - if the package has package identity do nothing.